src/scripts/parseCodeSystems.ts

```diff
@@ -0,0 +1,35 @@
+// Code systems source: https://cts.nlm.nih.gov/fhir/metadata?_format=json
+
+import { readFileSync, writeFileSync } from 'fs';
+import path from 'path';
+
+const inputPath = path.resolve(path.join(__dirname, '../code-system/cts-metadata.json'));
+const outputPath = path.resolve(path.join(__dirname, '../code-system/system-map.json'));
+const ctsMetadata = readFileSync(inputPath, 'utf8');
+
+/**
+ * Parse cts-metadata and transform into code system url->name mapping
+ */
+async function parse(codeSystemJSON: string) {
+  const codeSystemData = (await JSON.parse(codeSystemJSON)) as fhir4.CapabilityStatement;
```

Comment:
This function is not async https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#return_value

```suggestion
  const codeSystemData = JSON.parse(codeSystemJSON) as fhir4.CapabilityStatement;
```